cmd/services.rb

```diff
@@ -150,7 +150,7 @@ def run
           ::Service::Commands::Start.run(targets, args.file, verbose: args.verbose?)
         when *::Service::Commands::Stop::TRIGGERS
           max_wait = args.max_wait.to_f
-          ::Service::Commands::Stop.run(targets, verbose: args.verbose?, no_wait: args.no_wait?, max_wait:)
+          ::Service::Commands::Stop.run(targets, verbose: args.verbose?, no_wait: args.no_wait?, max_wait: max_wait)
```

Comment:
@albertterradas What Ruby version are you running? The existing syntax is valid on the version of Ruby that Homebrew requires.